transform/babel-plugin-fbt/bin/translate.js

```diff
@@ -208,5 +218,21 @@ if (argv[args.HELP]) {
 const output = argv[args.STDIN]
   ? processJSON(JSON.parse(fs.readFileSync(process.stdin.fd, 'utf8')))
   : processFiles(argv[args.SRC], argv[args.TRANSLATIONS]);
-const json = JSON.stringify(output, ...(argv[args.PRETTY] ? [null, ' '] : []));
-process.stdout.write(json);
+
+function createJSON(obj) {
+  return JSON.stringify(obj, ...(argv[args.PRETTY] ? [null, ' '] : []));
+}
+
+const splitByLocaleTo = yargs.argv[args.SPLIT_BY_LOCALE_TO];
+if (splitByLocaleTo) {
+  fs.mkdirSync(splitByLocaleTo, {recursive: true});
+
+  Object.keys(output).forEach(function(locale) {
```

Comment:
I'm also fine with `for (let locale in output) {...}`, but if you want to keep the functional style, use a lambda: `Object.keys(...).forEach(locale => {...})`

Reply:
Fixed in 27cc3d4